app/models/book.rb

```diff
@@ -8,4 +8,6 @@ class Book < ApplicationRecord
     ->(gender:) { where('lower(gender) like ?', "%#{gender.downcase}%") },
     ->(title:) { where('lower(title) like ?', "%#{title.downcase}%") }
   )
+  extend FriendlyId
```

Comment:
move to top of class 

read this https://github.com/rubocop/ruby-style-guide#consistent-classes